sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -540,8 +540,14 @@ object QueryCompilationErrors {
       s"rename temporary view from '$oldName' to '$newName': destination view already exists")
   }
 
-  def databaseNotEmptyError(db: String, details: String): Throwable = {
-    new AnalysisException(s"Database $db is not empty. One or more $details exist.")
+  def cannotDropNonemptyDatabaseError(db: String, details: String): Throwable = {
+    new AnalysisException(s"Cannot drop a non-empty database: $db. One or more $details exist.")
+  }
+
+  def cannotDropNonemptyNamespaceError(namespace: Seq[String]): Throwable = {
+    new AnalysisException(
+      s"Cannot drop a non-empty namespace: ${namespace.quoted}. " +
+        "Use CASCADE option to drop a non-empty namespace.")
```

Comment:
Move `cannotDropNonemptyNamespaceError` from `QueryExecutionErrors` to `QueryCompilationErrors` as [SPARK-33539](https://issues.apache.org/jira/browse/SPARK-33539)

Reply:
Does CASCADE work for the v1 command? If yes, we should also mention it in the v1 error message.

Reply:
Yes, it works. I'll mention it in v1 error message. Thank you, @cloud-fan!